data/json/items/armor.json

```diff
@@ -10200,7 +10200,7 @@
     "id": "backpack_hiking",
     "type": "ARMOR",
     "name": "hiking backpack",
-    "description": "A large sized hiking backpack excellent storage space.",
+    "description": "A large sized hiking backpack with plenty of storage space..",
```

Comment:
Double period .. here and in backpack_tactical_large desc. 😄

Reply:
Whoops lol